src/components/auth/LoginBox.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import './auth.css';
+import { sendRequest } from '../../sendRequest/sendRequest';
+import { Redirect } from 'react-router-dom';
+import { loginApi } from '../../sendRequest/apis';
+
+class LoginBox extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      loggedIn: false
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  submitLogin() {
+    sendRequest(loginApi, {
+      method: 'POST',
```

Comment:
the method, headers and body should all be apart of the sendRequest method that should have defaults be set there only.